src/constants.js

```diff
@@ -13,12 +13,4 @@ export const envs = {
 
 // Variables
 export const prefix = '++';
-export const validCmds = [
-  'ama',
-  'faq',
-  'pins',
-  'ping',
-  'salary',
-  'settings',
-  'perms'
-];
+export const validCmds = ['ama', 'faq', 'pins', 'ping', 'salary', 'settings'];
```

Comment:
we should probably normalize when to break up into separate lines.

Reply:
It's normalized - based on eslint + prettier